src/pages/en/core-concepts/astro-components.md

```diff
@@ -275,6 +275,8 @@ const { name } = Astro.props
 </Wrapper>
 ```
 
+This is a feature of Astro components - it can't be used when including a framework component inside a `.astro` file, even if that framework supports named slots.
```

Comment:
This bit is true, but the way its written it infers the wrong meaning, Astro uses the html `<template>` `<slot>` elements to pass through children into components. Its just that slots when in use in UI-land tend to be abstractions over this, hence the incompatibility between ui frameworks using `props.children`  and Astro using `<slots/>`